lms/djangoapps/grades/models.py

```diff
@@ -27,7 +27,7 @@
 
 # Used to serialize information about a block at the time it was used in
 # grade calculation.
-BlockRecord = namedtuple('BlockRecord', ['locator', 'weight', 'max_score'])
+BlockRecord = namedtuple('BlockRecord', ['locator', 'weight', 'r_possible', 'graded'])
```

Comment:
Since `BlockRecord` is defined somewhat farther from the documentation in `scores.py`, I think it's worth the extra 2 characters to label `raw_possible` explicitly.